packages/google-cloud-discoveryengine/google/cloud/discoveryengine_v1alpha/services/chunk_service/transports/rest.py

```diff
@@ -0,0 +1,726 @@
+            response = getattr(self._session, method)(
+                "{host}{uri}".format(host=self._host, uri=uri),
+                timeout=timeout,
+                headers=headers,
+                params=rest_helpers.flatten_query_params(query_params, strict=True),
+            )
+
+            # In case of error, raise the appropriate core_exceptions.GoogleAPICallError exception
+            # subclass.
+            if response.status_code >= 400:
+                raise core_exceptions.from_http_response(response)
+
+            # Return the response
+            resp = chunk.Chunk()
+            pb_resp = chunk.Chunk.pb(resp)
+
+            json_format.Parse(response.content, pb_resp, ignore_unknown_fields=True)
+            resp = self._interceptor.post_get_chunk(resp)
+            return resp
+
+    class _ListChunks(ChunkServiceRestStub):
+        def __hash__(self):
+            return hash("ListChunks")
+
+        __REQUIRED_FIELDS_DEFAULT_VALUES: Dict[str, Any] = {}
+
+        @classmethod
+        def _get_unset_required_fields(cls, message_dict):
+            return {
+                k: v
+                for k, v in cls.__REQUIRED_FIELDS_DEFAULT_VALUES.items()
+                if k not in message_dict
+            }
+
+        def __call__(
+            self,
+            request: chunk_service.ListChunksRequest,
+            *,
+            retry: OptionalRetry = gapic_v1.method.DEFAULT,
+            timeout: Optional[float] = None,
+            metadata: Sequence[Tuple[str, str]] = (),
+        ) -> chunk_service.ListChunksResponse:
+            r"""Call the list chunks method over HTTP.
+
+            Args:
+                request (~.chunk_service.ListChunksRequest):
+                    The request object. Request message for
+                [ChunkService.ListChunks][google.cloud.discoveryengine.v1alpha.ChunkService.ListChunks]
+                method.
+                retry (google.api_core.retry.Retry): Designation of what errors, if any,
+                    should be retried.
+                timeout (float): The timeout for this request.
+                metadata (Sequence[Tuple[str, str]]): Strings which should be
+                    sent along with the request as metadata.
+
+            Returns:
+                ~.chunk_service.ListChunksResponse:
+                    Response message for
+                [ChunkService.ListChunks][google.cloud.discoveryengine.v1alpha.ChunkService.ListChunks]
+                method.
+
+            """
+
+            http_options: List[Dict[str, str]] = [
+                {
+                    "method": "get",
+                    "uri": "/v1alpha/{parent=projects/*/locations/*/dataStores/*/branches/*}/documents/*/chunks",
+                },
+                {
+                    "method": "get",
+                    "uri": "/v1alpha/{parent=projects/*/locations/*/collections/*/dataStores/*/branches/*}/documents/*/chunks",
+                },
+            ]
```

Comment:
Unit tests are failing with this `uri`. If I change the segment below then tests pass.
```
"uri": "/v1alpha/{parent=projects/*/locations/*/dataStores/*/branches/*}/documents/*/chunks",
```
to
```
"uri": "/v1alpha/{parent=projects/*/locations/*/dataStores/*/branches/*/documents/*}/chunks",
```

I've filed b/328843326 to continue the investigation internally.